### ISSUE TITLE
Reorder columns long layout

### ISSUE THREAD

**Issue by gmcusaro** [ogham/exa#995]
Is there a way to choose and reorder columns in long layout?
Something like:
```
  - user
  - size
  - name
  - date
 ```

**Comment by ariasuni** [comment]
No, there’s no way to do that currently, and it’s blocked by #511.

**Comment by ariasuni** [comment]
Closing in favor of: https://github.com/eza-community/eza/issues/148
